paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -419,11 +419,11 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         }
     }
 
-    private fun buildPrimaryButtonText(): String? {
+    private fun buildPrimaryButtonText(): String {
         return when {
             args.isCompleteFlow -> {
                 if (args.clientSecret is PaymentIntentClientSecret) {
-                    args.formArgs.amount?.buildPayButtonLabel(application.resources)
+                    args.formArgs.amount!!.buildPayButtonLabel(application.resources)
```

Comment:
Not to be addressed here, but it would be nice to model this so we don't have to do the `is` check, and force unwrap the amount.

Reply:
Yep. We should strive to model the mode better (payment vs. setup) and potentially compute these values during initialization, for instance in `PaymentSheetLoader`.